CONTRIBUTORS

```diff
@@ -5,6 +5,6 @@
 #
 Alexander Fenster <fenster@google.com>
 Chris Wilcox <crwilcox@google.com>
-Christopher Wilcox <git@crwilcox.com>
+Jonathan Lui <kinwa91@msn.com>
```

Comment:
can we use Google email here (and on GitHub as well)?  Can be done by adding you to `.mailmap` and regenerating scaffolding.

Reply:
Yes please! I just updated my primary email on Github. Where shall I edit this `.mailmap`

Reply:
Looks like my one above was my google :) Is it pretty standard to change the default github and .git config to be @google addresses?